test/test_visualization.jl

```diff
@@ -143,6 +143,13 @@ isdir(outdir) && rm(outdir, recursive=true)
     end
   end
 
+  @testset "adapt_to_mesh_level" begin
+    @test_nowarn_debug trixi_include(@__MODULE__, joinpath(examples_dir(), "1d", "elixir_euler_blast_wave.jl"),
+                                     tspan=(0,0.1))
+    @test PlotData1D(adapt_to_mesh_level(sol, 4)...) isa PlotData1D
+    @test PlotData1D(adapt_to_mesh_level!(sol, 4)...) isa PlotData1D
+  end
```

Comment:
Do we have a guarantee like "Adapt to one level higher and projecting back does not change the solution"? I would expect that this is the case for our current L² based operators. If so, it might be nice to have a test for this (instead of or in addition to the smoke test above).

Reply:
Good idea! I tried to follow your suggestion in adabb726.